frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/JsonInputParam.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { TextArea } from '@patternfly/react-core';
+import { InputParamProps } from './types';
+
+export const JsonInputParam: React.FC<InputParamProps> = ({ id, value, onChange }) => {
+  const [jsonValue, setJsonValue] = React.useState(value ? JSON.stringify(value) : '');
+
+  return (
+    <TextArea
```

Comment:
Mocks mention having different UX for arrays

Reply:
What mocks are you referring to? Both struct and list are both just JSON. This was somewhat discussed and was agreed to use TextArea for JSON fields here: https://redhat-internal.slack.com/archives/C05U1Q749PV/p1709737988098779?thread_ts=1709664369.749849&cid=C05U1Q749PV

Reply:
the mocks in the jira: https://issues.redhat.com/secure/RapidBoard.jspa?rapidView=18687&view=detail&selectedIssue=RHOAIENG-2980&quickFilter=119321

i see @xianli123 said to just go with the text boxes for now. He can chime in for the UX review of this PR and confirm

Reply:
That screenshot is showing 2 states of the same thing. The first JSON input is showing the collapsed state, the 2nd is showing the expanded state.